test/spell/DisableV2Arb.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.28;
+
+import {DisableV2Common} from "./DisableV2Common.sol";
+
+/// @notice Arbitrum-specific spell that disables V2 permissions for JTRSY_USDC only
+contract DisableV2Arb is DisableV2Common {
+    address public constant JTRSY_VAULT_ADDRESS = address(0xe98Cf1221bC3F38D8bb132b8434A6F8885071173);
+
+    function getJTRSYVaultAddress() internal pure override returns (address) {
+        return JTRSY_VAULT_ADDRESS;
+    }
```

Comment:
Should not the above suffice as a getter? I mean. `DisabelV2Arb.JTRSY_VAULT_ADDRESS()`

BTW, is the method pure because returning a constant dont imply reading, right? That's curious and logical!

Reply:
> Q: Should not the above suffice as a getter? I mean `DisabelV2Arb.JTRSY_VAULT_ADDRESS()`

Unfortunately, that doesn't work. The virtual function pattern is actually necessary here because parent contracts can't directly access constants defined in child contracts because constants are resolved at compile time.

We could save 100-200 deployment gas if we didn't store the `address public constant JTRSY_VAULT_ADDRESS` but IMO it's cleaner this way and @hieronx approved it.

Reply:
Ok! Makes sense now I understand the hierarchy.

Reply:
> BTW, is the method pure because returning a constant dont imply reading, right? That's curious and logical!

Apparently, constants are actually not storage variables and are inlined in the used code paths at compile time ([ref](https://ethereum.stackexchange.com/a/140629)):

```solidity
// What you write:
function getJTRSYVaultAddress() internal pure override returns (address) {
    return JTRSY_VAULT_ADDRESS;
}

// What the compiler generates (conceptually):
function getJTRSYVaultAddress() internal pure override returns (address) {
    return address(0x1d01Ef1997d44206d839b78bA6813f60F1B3A970);
}
```